app/src/test/java/co/nimblehq/compose/crypto/ui/screens/home/HomeViewModelTest.kt

```diff
@@ -63,10 +66,39 @@ class HomeViewModelTest : BaseViewModelTest() {
             }
         }
 
+    @Test
+    fun `When getting trending coin list successfully, it should return my coin list`() =
+        runBlockingTest {
+            testDispatcher.pauseDispatcher()
```

Comment:
@luongvo, Regarding [this](https://github.com/nimblehq/jetpack-compose-crypto/pull/27#discussion_r963652707)  Have you tried to resolve this? Shall we leave todo here if we don't fix them in this PR 🙏

Reply:
@Wadeewee It takes time to migrate all to latest tech-stack, I think I will not spend much effort with it. I will migrate it in Android template instead. Here is the migration doc: https://github.com/Kotlin/kotlinx.coroutines/blob/master/kotlinx-coroutines-test/MIGRATION.md

Reply:
Got it @luongvo I think you just leave todo is good then 🚀